lightning/Cargo.toml

```diff
@@ -32,10 +32,6 @@ bitcoin = "0.26"
 hex = { version = "0.3", optional = true }
 regex = { version = "0.1.80", optional = true }
 
-[dev-dependencies.bitcoin]
-version = "0.26"
```

Comment:
Oops, we still need this block for `cargo test -p lightning` because that isn't `_test_util` just `test`

Reply:
Ah, thanks that's what I'm looking for